client/home.html

```diff
@@ -62,6 +62,10 @@
         var loadingBarHome = document.getElementById('loading-bar-home');
 
         var config = {};
+        var consts = {}
+        fn.request("api/get_const_vars", "GET", function (err, res) {
```

Comment:
This won't work this way. JS is asynchronous so you need to wait for this request to finish before continuing everything else, which is most definitely a near-impossible task utilizing the current client. 

I'd suggest adding the constants at "compile time" using a task runner and extending the npm build command definition in the `package.json`

Reply:
Server-side looks good, though 👍